client/lib/analytics/index.js

```diff
@@ -513,4 +295,228 @@ const analytics = {
 	},
 };
 emitter( analytics );
-export default analytics;
+
+const ga = {
```

Comment:
I'm a fan of inlined `export`s -- any reason for doing it [in bulk](https://github.com/Automattic/wp-calypso/pull/20247/files#diff-d088a91c46152a3ed739a47ed406c5bbR522)?

Reply:
No reason, I generally prefer inline myself. It's just the way it came out in this quick exploratory PR 😉